src/govpress-unit/code-review.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Code review in GovPress
+last_reviewed_at: ""
+---
+
+
+GovPress follows [the same process for software development](https://playbook.dxw.com/tech/how-we-do-development/) as the Tech Team. However, developers in the Tech Team work on a small number of long-running projects, with relatively large teams. In GovPress we work on a very large number of small projects, with small teams, so we have a slightly different emphasis on how we write code.
+
+In particular, if you are requesting review, you should be aware that reviewers will be reviewing multiple changesets over a large number of very different projects each day, and will not necessarily be aware of the full context of each story. This makes it especially important that we follow a consistent and efficient process.
+
+This document should therefore be read _in addition_ to the [Tech Team documentation](https://playbook.dxw.com/tech/how-we-do-development/) and applies to all PRs that are not automatically generated by services such as Dependabot or GovPress Tools.
+
+## Creating PRs
+
+When you create a PR the **Description** section of the PR should include, as a minimum:
+
+1. A full description of the story that the PR addresses. For example:
+    * [nao/353](https://github.com/dxw/nao/pull/353)
+    * [whippet/212](https://github.com/dxw/whippet/pull/212)
+    * [website/359](https://github.com/dxw/website/pull/359)
+1. Links to any relevant external documentation, which might be internal documentation (Zendesk, Trello, etc.) or external documentation. For example:
```

Comment:
It seems that the list numbers are back to 1, is that correct?

Reply:
Yes, they should be rendered automatically (the reason for making them 1. in the code is to make it easier for someone to reorder the list later on)

Reply:
Cool, thanks